drivers/sensor/apds9253/apds9253.c

```diff
@@ -131,7 +131,7 @@ static int apds9253_attr_set_gain(const struct device *dev, uint8_t gain)
 		return 0;
 	}
 
-	static uint8_t value_map[] = {
+static const uint8_t value_map[] = {
```

Comment:
Indentation is lost in the change.

Reply:
duh - of course! Thanks for paying attention